clustering/web/hotrod/src/main/java/org/wildfly/clustering/web/hotrod/session/HotRodSessionFactory.java

```diff
@@ -54,17 +51,21 @@
 import org.wildfly.clustering.web.cache.session.SessionAttributesFactory;
 import org.wildfly.clustering.web.cache.session.SessionCreationMetaDataEntry;
 import org.wildfly.clustering.web.cache.session.SessionMetaDataFactory;
-import org.wildfly.clustering.web.cache.session.SimpleSessionCreationMetaData;
+import org.wildfly.clustering.web.cache.session.SimpleSessionAccessMetaData;
 import org.wildfly.clustering.web.hotrod.logging.Logger;
 import org.wildfly.clustering.web.session.ImmutableSession;
 import org.wildfly.clustering.web.session.ImmutableSessionAttributes;
 import org.wildfly.clustering.web.session.ImmutableSessionMetaData;
 import org.wildfly.security.manager.WildFlySecurityManager;
 
 /**
+ * Factory for creating a {@link org.wildfly.clustering.web.session.Session} backed by a set of {@link RemoteCache} entries.
  * @author Paul Ferraro
+ * @param <C> the marshalling context type
+ * @param <V> the session attribute entry type
+ * @param <C> the local context type
```

Comment:
s/<C>/<L> - but we don't need to fix this now.

Reply:
Fixed.